oap-server/server-starter/src/main/resources/bydb.yml

```diff
@@ -52,15 +52,41 @@ groups:
   # The "normal" section defines settings for datasets not specified in "super".
   # Each dataset will be grouped under a single group named "normal".
   recordsNormal:
+    # The settings for the default "hot" stage.
     shardNum:  ${SW_STORAGE_BANYANDB_GR_NORMAL_SHARD_NUM:1}
     segmentInterval: ${SW_STORAGE_BANYANDB_GR_NORMAL_SI_DAYS:1}
     ttl: ${SW_STORAGE_BANYANDB_GR_NORMAL_TTL_DAYS:3}
+    # Enable the additional stages. The "hot" stage is always enabled by default.
+    enabledAdditionalStages: ${SW_STORAGE_BANYANDB_GR_NORMAL_ENABLED_ADDITIONAL_STAGES:"warm,cold"}
+    defaultQueryStages: ${SW_STORAGE_BANYANDB_GR_NORMAL_DEFAULT_QUERY_STAGES:"hot,warm"}
+    warm:
+      shardNum: ${SW_STORAGE_BANYANDB_GR_NORMAL_WARM_SHARD_NUM:1}
+      segmentInterval: ${SW_STORAGE_BANYANDB_GR_NORMAL_WARM_SI_DAYS:1}
+      ttl: ${SW_STORAGE_BANYANDB_GR_NORMAL_WARM_TTL_DAYS:3}
```

Comment:
warm TTL should be longer than default ttl, and cold should be longer than warm.

Reply:
This is general for metrics and records. One question, record normal should only have sampling data, right? I can't see much point to keep this data such long.